.github/workflows/SwiftFormat.yml

```diff
@@ -0,0 +1,76 @@
+name: SwiftFormat
+
+on:
+  workflow_dispatch:
+
+jobs:
+  windows:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
```

Comment:
I'm not super familiar with github actions but do we need the matrix here if we'll be building the same tag/branch on all architectures of the build?

Reply:
It is originally in @compnerd action because the plan is that this is built for an arbitrary number of toolchain releases when (a future version of) this is upstreamed to nick's SwiftFormat repo. Right now, the matrix is no-op.

Reply:
Makes sense!

Reply:
The matrix I was using was for the Swift compiler version.  Since we are not ABI stable on Windows, we need the release version matrix for the compile still as the standard library will change.

Reply:
@mangini did we test building with the 5.7 toolchain and running against a 5.9 toolchain?  I'm worried about the ABI instability in the runtime meaning that we won't be able to use the releases with newer toolchains.

Reply:
@compnerd Do you mean that this SwiftFormat binary is dynamically-linked and that its version needs to include the version of the swift compiler that compiled it? If so, I interpret it as that the matrix needs to have both the compiler version and the SwiftFormat version.

Reply:
@hjyamauchi correct.  `swiftCore`, `Foundation`, and `dispatch` are always dynamically linked, and the version needs to be the version from the runtime distributed with the toolchain that it was built with.  Extending the matrix to both sounds right.